src/main/resources/templates/layouts/baseLayout.html

```diff
@@ -29,6 +31,11 @@
 <script
     th:src="@{{cdnUrl}/javascripts/vendor/jquery-3.3.1.min.js(cdnUrl=${@environment.getProperty('cdn.url')})}"></script>
 <div class="entire-wrapper">
+  <div id="skiplink-container">
+    <div>
```

Comment:
Why is there a <div> inside an <div> here?

Reply:
Because that is how it is written on the current site. But it can be removed

Reply:
Please remove it if it serves no purpose.